hooks/useKakaoMap.ts

```
@@ -0,0 +1,66 @@
+      })
+    }
+
+    if (!document.getElementById(scriptId)) {
+      const script = document.createElement("script")
+      script.id = scriptId
+      script.src = `//dapi.kakao.com/v2/maps/sdk.js?appkey=${KAKAO_JAVASCRIPT_KEY}&autoload=false&libraries=services`
+      script.async = true
+      document.head.appendChild(script)
+      script.onload = initMap
+    } else {
+      if (window.kakao && window.kakao.maps) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)